lib/restforce.rb

```diff
@@ -45,6 +45,24 @@ module Tooling
   UnauthorizedError   = Class.new(Error)
   APIVersionError     = Class.new(Error)
 
+  # Inherit from Faraday::Error::ResourceNotFound for backwards-compatibility
+  # Consumers of this library that rescue and handle Faraday::Error::ResourceNotFound
+  # can continue to do so.
+  NotFoundError       = Class.new(Faraday::Error::ResourceNotFound)
+
+  # Inherit from Faraday::Error::ClientError for backwards-compatibility
+  # Consumers of this library that rescue and handle Faraday::Error::ClientError
+  # can continue to do so.
+  ResponseError       = Class.new(Faraday::Error::ClientError)
+  MatchesMultipleError= Class.new(ResponseError)
```

Comment:
The spacing looks a bit odd here

Reply:
Do you have a preference between me indenting everything further by one space or just removing the justify-alignment of `=`s in this diff?